client/src/test/java/pro/panopticon/client/sensor/impl/SuccessrateSensorTest.java

```diff
@@ -1,6 +1,5 @@
 package pro.panopticon.client.sensor.impl;
 
-import org.hamcrest.core.StringContains;
 import org.junit.Test;
```

Comment:
> En liten testenøtt for de som liker sånt. 

Hva med at man bruker Clock, og har et internt felt for den. Og så mocker eller overskriver man den i testene med nye klokker som gir ønskede tider når man kaller LocalDateTime.now(clock)?

Reply:
Går det eventuelt an å innføre en overloaded konstruktør og la den som finnes i dag implementere en default sånn tids-"provider"? Da endrer man ikke API-et for alle dagens brukere av sensoren, men man tillater samtidig testing. 

Noe sånt noe (syntaxen er garantert feil her altså, men konseptet er forhåpentlig tydelig):

```
    public SuccessrateSensor(int numberToKeep, Double warnLimit, Double errorLimit) {
        this(numberToKeep, warnLimit, errorLimit, new NowSupplier(){return LocalDateTime.now()})
    }    

    public SuccessrateSensor(int numberToKeep, Double warnLimit, Double errorLimit, NowSupplier nowSupplier) {
        this.numberToKeep = numberToKeep;
        this.warnLimit = warnLimit;
        this.errorLimit = errorLimit;
        this.nowSupplier = nowSupplier;
    }
```

Reply:
Kan også fjerne `public` fra den andre konstruktøren, så blir den ikke eksponert ut fra pakka :)

Reply:
Testet @hemm1 sin variant nå, ble egentlig veldig greit! https://github.com/nsbno/panopticon/pull/4/commits/2efe7b3ce765d87d1e21eea34431f48514c2c2f2